packages/react-core/src/components/DataList/examples/DataList.md

```diff
@@ -1125,6 +1125,10 @@ import {
 
 ### Draggable
 
+Draggable data lists used to have their own HTML5 API based drag and drop. Upon initial implementation we thought this implementation sufficient for all future drag and drop functionality. We were wrong.
```

Comment:
I think that this statement - "Upon initial implementation we thought this implementation sufficient for all future drag and drop functionality" is always true - and obviously often proven untrue. So I wonder if it'd be more constructive to say something like
"The initial implementation has been unable to fulfill all requirement requests."